### ISSUE TITLE
Custom build fails with npm 3

### ISSUE THREAD

**Issue by mlegenhausen** [openlayers/ol3#4221]
Cause of the introduction of a flat dependency hierarchy the path of jsdoc has changed.

The problem occurs in `tasks/generate-info.js:17`.

```
var jsdoc = path.join(__dirname, '..', 'node_modules', '.bin', 'jsdoc');
```

needs to be changed to

```
var jsdoc = path.join(__dirname, '..', '..', '.bin', 'jsdoc');
```

to work correctly with npm3.

Error message:

```
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn /Users/mlegenhausen/dev/werk85/dk-navigator-app/node_modules/openlayers/node_modules/.bin/jsdoc ENOENT
    at exports._errnoException (util.js:849:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:430:9)
    at process._tickCallback (node.js:344:17)
```

**Comment by tschaub** [comment]
Thanks for the report @mlegenhausen.  Let me know if the change in #4347 resolves the issue for you.  If you have a `package.json` with `openlayers` as a dependency, you should be able to test it out by updating your dependency (temporarily) to look like this:
```
  "openlayers": "git://github.com/tschaub/ol3.git#dc0e37b7823f200c674fb8d5934fd23eba9d2340"
 ```

**Comment by mlegenhausen** [comment]
Does this version already contain the fix for node 4? Got the same error as in #4163

**Comment by tschaub** [comment]
@mlegenhausen - the fixes are in separate branches.  So the commit referenced above should work with npm@3 but node < 4.  When I get that merged, I'll rebase the `node-four` branch so it will contain both fixes.
